relayer-cli/src/commands/tx/client.rs

```diff
@@ -214,6 +215,14 @@ pub struct TxUpgradeClientCmd {
         help = "Identifier of the client to be upgraded"
     )]
     client_id: ClientId,
+
+    #[clap(
+        long = "upgrade-height",
+        required = true,
+        value_name = "SRC_UPGRADE_HEIGHT",
+        help = "The height at which the source chain should halt in order to perform the client upgrade"
+    )]
+    target_upgrade_height: Height,
```

Comment:
Let's use `u64` here instead of `Height`; it's easier for users and more consistent with the rest of the cli